code/modules/clothing/outfits/ert.dm

```diff
@@ -473,8 +473,8 @@
 	if(!istype(mod))
 		return
 	var/obj/item/mod/module/hat_stabilizer/hat_holder = locate() in mod.modules
-	var/obj/item/clothing/head/helmet/space/beret/beret = new(hat_holder)
-	hat_holder.attached_hat = beret
+	var/obj/item/clothing/head/helmet/space/helm = new(hat_holder)
```

Comment:
If I wasn't already convinced you're making a troll pr, this definitely would be a good hint.